src/conf_mode/snmp.py

```diff
@@ -166,6 +166,10 @@ def verify(snmp):
             if 'community' not in trap_config:
                 raise ConfigError(f'Trap target "{trap}" requires a community to be set!')
 
+    if 'oid_enable' in snmp:
+        if 'all' in snmp['oid_enable']:
+            Warning(f'All oids are enabled and may lead to system instability and high resource consumption')
```

Comment:
Why only for `all`? I expect to see the warning message for any OID from RESTRICTED view.